docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -176,6 +178,48 @@ image:images/bar-chart-tutorial-2.png[Bar chart with sample logs data]
 . Click *Save and return*.
 
 
+[float]
```

Comment:
When we documented the new charts library in 7.12, we decided the docs should live only in the [Visualization section on the Advanced Settings page](https://www.elastic.co/guide/en/kibana/7.12/advanced-options.html#kibana-visualization-settings) since data analysts are typically unfamiliar with the Advanced settings and don't always have access.

Reply:
I think it would be ok to put the description of what's in the new charts library in Advanced Settlings.  We've done something similar with the "Use classic table" setting in Discover.  That way, the callout would require only one link.

### Callout

**User has permission**

You are using the legacy charts library, which will be removed in 7.16. Switch to the new library in [Advanced Settings](https://www.elastic.co/guide/index.html).


**User without permission**

You are using the legacy charts library, which will be removed in 7.16. Contact your system administrator to switch to the new library. 

### Advanced Settings

**XY axis legacy charts library**
The visualize editor uses a new charts library with improved performance, color palettes, fill capacity, and more. Enable this option if you prefer to use the legacy charts library.

**Pie legacy charts library**
The visualize editor uses new pie charts with improved performance, color palettes, label positioning, and more. Enable this option if you prefer to use to the legacy charts library.